### PR TITLE
fix(a11y): improve Lighthouse accessibility score

### PR DESCRIPTION
**Pull request by MrBrain295**
## Summary
Add names to buttons. Relates to #7692

### Problem
The buttons don't have names.

### Solution
Add names.

### CODE REVIEW

**Review by caugner — CHANGES_REQUESTED** [review]
Thank you for your PR. 🙏 

According to https://dequeuniversity.com/rules/axe/4.4/button-name (which Google redirects to), it probably should be a

> Non-empty `aria-label` attribute.

The "Buttons have an accessible name" rule is a bit misleading.

File: client/src/ui/organisms/article-actions-container/index.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const ArticleActionsContainer = ({
           extraClasses="sidebar-button"
           icon="sidebar"
           type="action"
+          name="Expand sidebar"
```
Comment (MrBrain295) [inline client/src/ui/organisms/article-actions-container/index.tsx:29]:
```suggestion
          aria-label="Expand sidebar"
```

File: client/src/ui/organisms/article-actions/index.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ export const ArticleActions = ({
         >
           <Button
             type="action"
+            name="Article actions"
```
Comment (MrBrain295) [inline client/src/ui/organisms/article-actions/index.tsx:56]:
```suggestion
            aria-label="Article actions"
```

**Comment by MrBrain295** [comment]
I can’t find the code for the last button Lighthouse is having issues with.

**Comment by caugner** [comment]
> I can’t find the code for the last button Lighthouse is having issues with.

It's this one (on mobile, when you toggle the sidebar, the rest of the page gets a backdrop, which hides the sidebar again):

https://github.com/mdn/yari/blob/587c1840af3c7a5fbd25059b199e80184f5edbf6/client/src/document/organisms/sidebar/index.tsx#L46-L50

**Review by caugner — CHANGES_REQUESTED** [review]
Let's also fix the last remaining button, see: https://github.com/mdn/yari/pull/7733#issuecomment-1337402060

File: client/src/document/organisms/sidebar/index.tsx [hunk 3]
```diff
@@ -47,6 +47,7 @@ export function SidebarContainer({ doc, children }) {
           extraClasses="backdrop"
           type="action"
           onClickHandler={() => setIsSidebarOpen(!isSidebarOpen)}
+          aria-label="Expand sidebar"
```
Comment (MrBrain295) [inline client/src/document/organisms/sidebar/index.tsx:50]:
```suggestion
          aria-label="Collapsesidebar"
```

**Review by caugner — APPROVED** [review]

**Pull request merged**